.env.example

```diff
@@ -3,7 +3,9 @@ ENVIRONMENT="staging"
 
 # URLs & Paths
 DEFAULT_SITE_URL=""
+# URL that will be prefixed to the asset URL - commonly the root URL
```

Comment:
- Not immediately apparent which variable this comment is referring to. Needs better code org so that the comment’s context is immediately apparent.
- Include some examples – they always make life easier
- The first part of the comment text is just a more verbose way of saying asset_base_url. We should use the comment to say what the variable does not. Eg: Full URL to the folder with the static assets as well as media assets. Typically same as the site url on dev, and CDN base url on production.

Reply:
Please see if it looks better now.